resources/views/partials/error.blade.php

```diff
@@ -0,0 +1,6 @@
+@if($errors->any())
+<div class="p-2 bg-dark text-light text-center">
+  <h3>{{ $errors->first() }}
+    <h3>
```

Comment:
this h3 isn't closed
```suggestion
    </h3>
```